jetty-osgi/test-jetty-osgi/src/test/java/org/eclipse/jetty/osgi/test/TestOSGiUtil.java

```diff
@@ -124,15 +138,24 @@ public static Option optionalRemoteDebug()
         {
             res.add(systemProperty("org.ops4j.pax.url.mvn.settings").value(System.getProperty("settingsFilePath")));
         }
+        
+        //sort out logging level from jetty
+        String jettyLogLevel = System.getProperty("org.eclipse.jetty.LEVEL", "WARN");
+        res.add(systemProperty("org.eclipse.jetty.LEVEL").value(jettyLogLevel));
```

Comment:
Why can't this use a `jetty-logging.properties` file instead?

Reply:
It seemed more convenient to be able to run the test and pass in the log level on the command line of the test, so it is very quick to change log levels. Is there some advantage in using a jetty-logging.properties file instead?

Reply:
Everywhere else, if we want to change logging levels, we look for a `jetty-logging.properties` file and make the change, controlling more then 1 level, message escaping, etc as well.

For jetty-osgi, we have to remember to edit a source file that's 3 levels of indirection away from the problem we are working to resolve.

We used to be able to edit a log4j.xml in jetty-osgi to manage logging, but you removed that file in this PR.

Reply:
Log4j doesn't seem to work any more in the pax exam environment combined with jetty logging that now requires the slf4j interface, so the combination you see is the only combination I could get to work. I'll try creating a jetty-logging.properties file, which I assume the jetty logging infrastructure will take care of reading and parsing? Ie, all I have to do is provide it?

Reply:
Correct, `jetty-slf4j-impl` will load it.

https://github.com/eclipse/jetty.project/blob/jetty-10.0.x/jetty-slf4j-impl/src/main/java/org/eclipse/jetty/logging/JettyLoggerConfiguration.java#L169-L194

It essentially does ...

``` java
ClassLoader.getResource("jetty-logging.properties");
```

to find the properties URL and then load it from that URL.

Reply:
@joakime ok, I've modified the osgi tests so that they can use src/test/resources/jetty-logging.properties.  This is better because it gives more flexibility in setting the logging levels, but also means the chance of someone committing their changes for local testing to jetty-logging.properties is high.